app/views/home/download.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="heading-large">Preview and download your code</h1>
```

Comment:
I'd expect this story to be in the user journey, and to bring in all the required context (defining attributes for the picker) as setup for the next story. The next story should just be plugging in and pulling out.